prog/download_boot_image.rb

```diff
@@ -23,6 +24,10 @@ def default_boot_image_version(image_name)
     Config.send(config_name)
   end
 
+  def download_from_r2?
+    frame["download_r2"] || Config.is_e2e
```

Comment:
At some point, we should consider removing the `|| Config.is_e2e`, but it seems fine if it is temporary.